vars/Ubuntu-18.04.yml

```diff
@@ -1,4 +1,5 @@
 php_version: "{{ atom_php_version|default('7.2') }}"
+php_service_name: "php{{ php_version }}-fpm"
```

Comment:
@scollazo  - All looks correct to me - one question, should this change also be made to **vars/Ubuntu-16.04.yml** or is that considered unlikely to come up?

Reply:
Good catch @DavidHume-ArtefactualSystems . I agree the new `php_service_name` variable should be defined in xenial and trusty vars files for backwards compatibility because we are using the master version for this role instead of a dedicated branch for each version ( for instance stable/2.Y.x )

Reply:
I didn't add those knowingly, and thought about removing them from the qa/ branch, as the only supported versions for 2.6 are Redhat/Centos 7 and Ubuntu 18.04.

As the change seems simple enough, I followed your suggestions and added the new variable to Ubuntu-16.04.yml, but there is still an open question about removing Ubuntu-14.04.yml from here.

Reply:
For Ubuntu 14 we always can define it as extravariable, so looks good to me.